austrakka/utils/options.py

```diff
@@ -9,150 +8,140 @@
 from austrakka.utils.misc import AusTrakkaCliOption
 
 
-def _default_option_params(**default):
```

Comment:
What's causing this to clobber options in other usages? Doesn't usages like line 22 create a new dictionary?